.github/workflows/backend-tests.yml

```diff
@@ -2,13 +2,33 @@ name: Backend Tests
 
 on:
   push:
-    branches: [ main ]
+    branches: [main]
     paths:
-      - 'backend/**'
+      - "backend/**"
   pull_request:
-    branches: [ main ] 
+    branches: [main]
     paths:
-      - 'backend/**'  
+      - "backend/**"
+  workflow_dispatch:
+    inputs:
+      logLevel:
+        description: 'Log level'
+        required: true
+        default: 'warning'
+        type: choice
+        options:
+        - info
+        - warning
+        - debug
+      tags:
+        description: 'Test scenario tags'
+        required: false
+        type: boolean
+      environment:
```

Comment:
can we have example of values here?

Reply:
I added that for testing purposes. Will likely remove it before merge